Neos.Flow/Classes/Mvc/Dispatcher.php

```diff
@@ -110,7 +110,9 @@ public function dispatch(RequestInterface $request, ResponseInterface $response)
                 } else {
                     $securityLogger->info(sprintf('Starting authentication with entry point of type "%s"', get_class($entryPoint)), LogEnvironment::fromMethodName(__METHOD__));
                 }
-                $securityContext->setInterceptedRequest($request->getMainRequest());
+                if (!$token instanceof SessionlessTokenInterface) {
```

Comment:
```suggestion
                // TODO: We should only prevent storage of intercepted request in the session here, but we don't have a different storage mechanism yet.
                if (!$token instanceof SessionlessTokenInterface) {
```

A quick draft of a code-comment... not very elaborated, sorry :/